var/spack/repos/builtin/packages/qb3/package.py

```diff
@@ -18,3 +18,20 @@ class Qb3(CMakePackage):
 
     depends_on("cmake@3.5:", type="build")
     depends_on("libicd")
+
+    @property
+    def libs(self):
+        # Override because libs have different case than Spack package name
+        name = "libQB3*"
+        # We expect libraries to be in either lib64 or lib directory
+        # Try lib64 first
+        root = self.prefix.lib64
+        liblist = find_libraries(name, root=root, shared=True, recursive=False)
+        if liblist:
+            # Found libs in lib64, so return those
+            return liblist
+        else:
+            # Did not find libs in lib64, so try lib w/out 64 suffix
+            root = self.prefix.lib
+            liblist = find_libraries(name, root=root, shared=True, recursive=False)
+            return liblist
```

Comment:
Apologies I didn't see the comments earlier. Logic looks good, I'd shorten the code to something like:
```suggestion
        for root in (self.prefix.lib64, self.prefix.lib):
            liblist = find_libraries(name, root=root, shared=True, recursive=False)
            if liblist:
                break
        return liblist
```
Beware of typos, since I just used the web-editor and didn't actually try the suggestion above :slightly_smiling_face:

Reply:
Ping @payerle